### PR TITLE
refactor: merge cost-analysis-enhancement branch

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/dashboards/DashboardsContainer.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@ const { breadcrumbs } = useBreadcrumbs();
             <router-view />
         </template>
     </vertical-page-layout>
-    <centered-page-layout v-else-if="$route.meta.centeredLayout"
+    <centered-page-layout v-else-if="route.meta.centeredLayout"
```
Comment (piggggggggy) [inline apps/web/src/services/dashboards/DashboardsContainer.vue:29]: praise: 👍👍

File: apps/web/src/services/dashboards/shared/dashboard-widget-container/DashboardWidgetContainer.vue [hunk 2]
```diff
@@ -191,6 +189,39 @@ onMounted(async () => {
...
 
+<template>
+    <div ref="containerRef"
+         class="dashboard-widget-container"
+    >
+        <template v-if="!dashboardDetailState.loadingDashboard">
+            <template v-for="(widget) in reformedWidgetInfoList">
+                <component :is="widget.component"
+                           :id="widget.widget_key"
+                           :key="widget.widget_key"
```
Comment (piggggggggy) [inline apps/web/src/services/dashboards/shared/dashboard-widget-container/DashboardWidgetContainer.vue:200]: nitpick: In `vue3`, `<template>` must have `key`, not child component.
```suggestion
            <template v-for="(widget) in reformedWidgetInfoList"
                              :key="widget.widget_key"
            >
                <component :is="widget.component"
                           :id="widget.widget_key"
```

**Review by piggggggggy — APPROVED** [review]
LGTM! I left some comments.
You're so fast..

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisGroupByFilter.vue [hunk 3]
```diff
@@ -15,37 +26,127 @@ const costAnalysisPageState = costAnalysisPageStore.$state;
...
 <template>
     <div class="cost-analysis-group-by-filter">
-        <b class="label">{{ t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.GROUP_BY') }}</b>
-        <p-select-button v-for="groupByItem in state.allGroupByItems"
-                         :key="groupByItem.name"
-                         :value="groupByItem"
+        <b
+            class="label"
+        >
+            {{ $t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.GROUP_BY') }}</b>
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisGroupByFilter.vue:109]:
```suggestion
            {{ t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.GROUP_BY') }}</b>
```

**Pull request merged**